### ISSUE TITLE
Allow path completion to accept a completion item with the directory separator

### ISSUE THREAD

**Issue by Therzok** [nushell/nushell#6340]
### Related problem

```nu
cd ./dir/<TAB>

# this one is highlighted

**./dir/a/** 
./dir/b/
./dir/c/
```

Pressing `/` to complete does not work, seems like only Enter is supported.

### Describe the solution you'd like

Completion should commit when `/` or `\` on windows is pressed, as that would continue the muscle memory flow for what would normally be writing directory segments.

### Describe alternatives you've considered

I've read up on adding keybindings, but it's still not clear from any of the docs how to make the completion commit on a specific key.

### Additional context and details

I'm okay with this being closed as by design, but having some guidance on how to write such a plugin would be great! Thank you, I've been having a wonderful experience with `nu` over the past months.

**Comment by sholderbach** [comment]
Good point! That might require some cooperation on the reedline side to enable the transitive/conditional use as a keybinding.
